st2actions/st2actions/workflows/workflows.py

```diff
@@ -40,10 +47,15 @@
     queues.WORKFLOW_ACTION_EXECUTION_UPDATE_QUEUE,
 ]
 
+WORKFLOW_ENGINE = "workflow_engine"
+SHUTDOWN_ROUTINE = "shutdown_routine"
```

Comment:
Since this name is used for locking during startup and shutdown, please rename this to `WORKFLOW_ENGINE_START_STOP_SEQ = "workflow_engine_start_stop_seq"`

Reply:
Addressed